owners/src/github.js

```diff
@@ -25,9 +25,19 @@ class PullRequest {
    * @param {!string} author username of the pull request author.
    * @param {!string} headSha SHA hash of the PR's HEAD commit.
    * @param {!string} description pull request description.
+   * @param {!string} state pull request status.
    */
-  constructor(number, author, headSha, description) {
-    Object.assign(this, {number, author, headSha, description});
+  constructor(number, author, headSha, description, state) {
+    Object.assign(this, {number, author, headSha, description, state});
+  }
+
+  /**
+   * Whether or not the pull request is open.
+   *
+   * @return {boolean} true if the pull request is open.
+   */
+  get isOpen() {
+    return this.state.toLowerCase() === 'open';
```

Comment:
It does not, that's a bit of a different challenge to address, since that's not really stateful. Ie, currently we trigger reviewer assignment on "new" PRs, to avoid adding comments and reviewers after new commits or reviewer changes. Draft vs. "real" PR is a different distinction, and I need to figure out the best way to discern between "existing PR with new commits" versus "previously draft PR that is no longer a draft"